src/default.js

```diff
@@ -73,6 +73,9 @@ Vue.component('drop', Drop);
                 redirect : to => arguments[0].navItems[0].route
             }];
 
+            // inject custom config into vuex base state
+            store.state.config.auth = config.data.auth
```

Comment:
ah - config.data is a string - this is not yet parsed json .... because of that auth does not work ...

Reply:
local issue with my config.json -> ignire